Lookup Time-in-Status Metrics

Manisharbaan opened the pull request.

Manisharbaan commented:
The bot will now prompt the user to enter the correct name or key. This has been fixed by adding a condition to the compound action, and we handled it via code itself. It's working as expected now, and I've also included a screenshot for your reference @Sudheer2137 

<img width="907" height="653" alt="image" src="https://github.com/user-attachments/assets/3317f8fc-35b7-441b-8c0e-7efd9fc94c66" />

Sudheer2137 commented:
@Manisharbaan can’t we format the 'Setting Up Time Tracking' section like the attached screenshot? Try using AI tools to find the best way to present clear steps so that customers can easily follow them and make the necessary changes.


<img width="759" height="470" alt="Screenshot 2025-08-21 at 11 37 16 AM" src="https://github.com/user-attachments/assets/e62c7cef-e25c-405b-9729-09a4ae11e42c" />

Manisharbaan commented:
<img width="1005" height="544" alt="image" src="https://github.com/user-attachments/assets/dc824d69-fc3b-4c08-b269-a98cf8dfc6ac" />
Kindly let me know if it is okay to proceed with setting up the tracking section so that I can raise the PR again ?
 @Sudheer2137

Sudheer2137 commented:
@Manisharbaan it looks good. You can proceed with updating the PR.

Sudheer2137 commented:
@Manisharbaan Can we unbold "Agent Studio" in the introduction, and is the JQL at the API endpoint correct as expected (pfa ss) ?
<img width="1503" height="593" alt="Screenshot 2025-08-25 at 1 24 39 PM" src="https://github.com/user-attachments/assets/e0ec9881-0afd-4e32-bb47-72d127d7c1e5" />

Manisharbaan commented:
I’ve updated the document with the new working endpoints, as the previous one was deprecated. Also, I’ve removed the bold formatting for 'Agent Studio' in the Introduction section. @Sudheer2137

Manisharbaan commented:
Thanks for the feedback. I've now included more natural and commonly used utterances as suggested. I've also used the provided examples for this use case and will follow the same approach for future ones. @Sudheer2137
